src/components/uploadpost/UploadPost.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { toast, ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { useNavigate } from 'react-router-dom';
+
+import {
+  StyleFormConatiner,
+  StyleInput,
+  CancelButton,
+  DoneButton,
+  LargeInput,
+  StyledLabel,
+  InputRow
+} from './StyledUpload.js';
+import { supabase } from '../../../supabase/supabase.js'; // supabase 사용하려면
+import SelectCategory from './../SelectCategory/SelectCategory';
+import { useDispatch, useSelector } from 'react-redux';
+
+const UploadPost = () => {
+  const [title, setTitle] = useState('');
+  const [nickname, setNickname] = useState('');
+  const [passwords, setPassword] = useState('');
+  const [content, setContent] = useState('');
+  const [time, setTime] = useState('');
+  const [participants, setParticipants] = useState('');
+  const [coverImage, setCoverImage] = useState('');
+  const [imageUrl, setImageUrl] = useState('');
+
+  const category = useSelector((state) => state.UploadPostSlice.category);
+  const lng = useSelector((state) => state.mapSlice.lng);
+  const lat = useSelector((state) => state.mapSlice.lat);
+
+  const navigate = useNavigate();
+
+  const handleClickConfirm = async () => {
+    let imageUploadUrl = null;
+
+    if (coverImage) {
+      imageUploadUrl = await uploadImage(coverImage);
+      console.log('Uploaded image URL:', imageUploadUrl);
+    }
+
+    const newObject = {
+      time: time,
```

Comment:
```suggestion
      time,
```
이렇게 key와 value가 같다면 축약도 가능합니다!